pkg/endpointmanager/manager.go

```diff
@@ -449,15 +450,25 @@ func GetPolicyEndpoints() map[policy.Endpoint]struct{} {
 }
 
 // AddEndpoint takes the prepared endpoint object and starts managing it.
-func AddEndpoint(ep *endpoint.Endpoint, reason string) (err error) {
+func AddEndpoint(owner regeneration.Owner, ep *endpoint.Endpoint, reason string) (err error) {
```

Comment:
The `owner` is part of `ep` since #8586, no need to pass it separately anymore :)

Reply:
@gandro it's a private field of endpoint we need to access it in the `AddEndpoint` function to send the monitor event.

Reply:
Oh, missed the fact that its not accessible. Sorry for the noise then!